Source/DafnyStandardLibraries/src/Std_Concurrent/Std_Concurrent.go

```diff
@@ -58,79 +68,152 @@ func (_this *MutableMap) ParentTraits_() []*_dafny.TraitID {
 }
 var _ _dafny.TraitOffspring = &MutableMap{}
 
-func (_this *MutableMap) Ctor__() {
+func (_this *MutableMap) Ctor__(bytesKeys bool) {
   {
+    _this.bytesKeys = bytesKeys
   }
 }
+
+func dafnyBytesToString(b interface{}) string {
+  return string(_dafny.ToByteArray(b.(_dafny.Sequence)))
+}
+func dafnyBytesFromString(b interface{}) _dafny.Sequence {
```

Comment:
Does that work for both unicode cases?

Reply:
Ah that's totally irrelevant because only Dafny `seq<byte>` values are involved here, no Dafny `string` values. I'm just using Go strings as comparable arbitrary bytes holders. Added a lot more comments as this is definitely not obvious.